jdr/jboss-as-jdr/src/main/java/org/jboss/as/jdr/JdrRunner.java

```diff
@@ -202,4 +208,48 @@ private String constructUri(final String protocol, final String host, final int
         // String the leading '//' if there is no protocol.
         return protocol == null ? uri.toString().substring(2) : uri.toString();
     }
+
+    private String obtainProductName() throws OperationFailedException {
+        try {
+            ModelNode operation = Operations.createReadAttributeOperation(new ModelNode().setEmptyList(), PRODUCT_NAME);
+            operation.get(INCLUDE_RUNTIME).set(false);
+            ModelControllerClient client = env.getClient();
+
+            if (client == null) {
+                client = env.getCli().getCommandContext().getModelControllerClient();
+            }
+
+            ModelNode result = client.execute(operation);
+
+            if (Operations.isSuccessfulOutcome(result)) {
+                return Operations.readResult(result).asString();
+            }
+
+            return null;
+        } catch (IOException e) {
+            return null;
```

Comment:
This doesn't look right. If there is a reason not to log an exception and rethrow it **must** be explicitly noted in comment.